py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -73,6 +73,9 @@ def get_shared_desi_proc_parser():
     parser.add_argument("--timingfile", type=str, help='save runtime info to this json file; augment if pre-existing')
     parser.add_argument("--no-xtalk", action="store_true", help='diable fiber crosstalk correction')
     parser.add_argument("--system-name", type=str, help='Batch system name (cori-haswell, perlmutter-gpu, ...)')
+    parser.add_argument("--extract-size", type=int, default=22, help="Size to use for GPU extract subcomm")
```

Comment:
Let's try to find a more descriptive name for this; at first glance I thought that "extract-size" referred to the extraction algorithm itself (e.g. something about the divide and conquer or number of wavelength bins or something) rather than something about the MPI communicator size.

Reply:
`--extract-subcomm-size` ?

Reply:
> `--extract-subcomm-size` ?

Done.